docs/en/pid-eaa-data-model.rst

```diff
@@ -537,6 +540,501 @@ The combined format for the PID issuance is represented below:
+    * - **eu.europa.ec.eudiw.pid.it.1**
+      - **tax_id_code**
+      - *tstr (text string)*. See :ref:`PID Claims fields Section <sec-pid-user-claims>`.
+
+
+Mobile Security Object
+----------------------
+
+The **issuerAuth** MUST contain a `Mobile Security Object` which is an untagged value containing a `COSE Sign1 Document`, as defined in `RFC 9052 - CBOR Object Signing and Encryption (COSE): Structures and Process <https://www.rfc-editor.org/rfc/rfc9052.html>`_. It MUST be composed by:
+
+* protected header
+* unprotected header
+* payload
+* signature.
+
+The **protected header** MUST contain the following parameter encoded in CBOR format:
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Element**
+      - **Description**
+      - **Reference**
+    * - **Signature algorithm**
+      - `-7` means ES256, SHA-256. 
+      - RFC8152
+
+.. note::
+    
+    Only the Signature Algorithm MUST be present in the protected headers, other elements SHOULD not be present in the protected header.
+
+
+The **unprotected header** MUST contain the following parameter:
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Element**
+      - **Description**
+      - **Reference**
+    * - **x5chain**
+      - Identified with the label 33
+      - `RFC 9360 CBOR Object Signing and Encryption (COSE) - Header Parameters for Carrying and Referencing X.509 Certificates <RFC 9360 CBOR Object Signing and Encryption (COSE) - Header Parameters for Carrying and Referencing X.509 Certificates>`_.
+
+.. note::
+    The `x5chain` is included in the unprotected header with the aim to make the Holder able to update the X.509 certificate chain, related to the `Mobile Security Object` issuer, without breaking the signature.
+
+The **payload** MUST contain the *MobileSecurityObject*, without any `content-type` definition, encoded as a *byte string* (bstr) with *CBOR Tag* 24.
+
+The `MobileSecurityObjectBytes` MUST have the following attributes:
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Element**
+      - **Description**
+      - **Reference**
+    * - **docType**
+      - See :ref:`Table <table-mdoc-documents-attributes>`.
+      - [ISO 18013-5#9.1.2.4]
+    * - **version**
+      - See :ref:`Table <table-mdoc-attributes>`.
+      - [ISO 18013-5#9.1.2.4]
+    * - **validityInfo**. 
+      - Object containing issuance and expiration datetimes. It MUST contain the following sub-value:
+
+          * *signed*
+          * *validFrom*
+          * *validUntil*
+      - [ISO 18013-5#9.1.2.4]
+    * - **digestAlgorithm**
+      - According to the algorithm defined in the protected header.
+      - [ISO 18013-5#9.1.2.4]
+    * - **valueDigests**
+      - Mapped digest by unique id, grouped by namespace.
+      - [ISO 18013-5#9.1.2.4]
+    * - **deviceKeyInfo**
+      - It MUST be set to the public key that is generated by the Wallet Instance. It contains the following sub-values.
```

Comment:
```suggestion
      - It MUST contain the Wallet Instance's public key contains the following sub-values.
```

Reply:
if we say that contains the pkey we then cannot say that contains sub attributes, a rewording is then required.